routes/articles.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const methodOverride = require('method-override');
+const articlesModule = require('../db/articles.js');
+
+let htmlContent = {};
+let error = { message : undefined };
+const router = express.Router();
+
+router.use(bodyParser.urlencoded({extended: true}));
+router.use(methodOverride('_method'));
+
+router.route('/')
+  .post((req, res) => {
+    let result = articlesModule.post(req.body);
+    if (result.success){
+      res.redirect('/articles/');
+    } else {
+      res.redirect('/articles/new');
+    }
+  })
+  .get((req, res) => {
+    htmlContent.articles = articlesModule.getArticles();
+    res.render('templates/articles/index', htmlContent);
+  });
+
+router.route('/new*')
```

Comment:
You shouldn't be using a * here. Then routes like /newlakdsjflakjsd are valid endpoints.